rust/pact_verifier/src/messages.rs

```diff
@@ -42,11 +43,12 @@ pub async fn verify_message_from_provider<F: RequestFilterExecutor>(
   };
   match make_provider_request(provider, &message_request, options, client).await {
     Ok(ref actual_response) => {
+      log::debug!("actual response = {:?}", actual_response);
```

Comment:
I think `response` might be a bit generic - should we use "internal payload" or something? Otherwise in some cases the logs are going to say "actual response: { ....}" and then "actual message { .... }" which will only be different if you have metadata.

(aside - maybe we should log metadata to `debug` and this whole payload to `trace`, since the message is already logged below?)

Reply:
I'm going to move this into the `make_provider_request` method alongside the other debugging statements. That way it will be logged for all uses of this important method.